tests/e2e/tests/regression/browser/onboarding.e2e.ts

```diff
@@ -29,9 +39,16 @@ fixture `Onboarding new user tests`
         await common.reloadPage();
     })
     .afterEach(async() => {
+        await cliPage.sendCommandInCli(`DEL ${indexName}`);
         await deleteDatabase(ossStandaloneConfig.databaseName);
     });
-test
+// https://redislabs.atlassian.net/browse/RI-4070, https://redislabs.atlassian.net/browse/RI-4067
+test.before(async() => {
+    await acceptTermsAddDatabaseOrConnectToRedisStack(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+    await cliPage.sendCommandInCli('flushdb'); // to delete all indexes in order not to fail FT.INFO ${indexName}
```

Comment:
flushdb deletes not only indexes but all keys in db as well
if we add parallelization in future it wil break all tests

to delete all indexes you can use
FT._LIST + FT.DROPINDEX

Reply:
or you can ask Artem to create and add docker container with a new database without keys and indexes (but with plugins) for such tests